### ISSUE TITLE
Is the test accuracy given by the training process the final result?

### ISSUE THREAD

**Issue by chopine** [yjxiong/caffe#94]
Hi, @yjxiong. I used your code to train the temporal vggnet-16 on ucf101 split1, but after training, the test accuracy was only  0.768, which is lower than 0.857, the result you gave in your article.
Here is the last several lines given by the training process:

I0728 00:22:48.208994 24418 solver.cpp:424] Iteration 30000, Testing net (#0)
I0728 00:22:59.593644 24418 solver.cpp:481]     Test net output #0: accuracy = 0.768236
I0728 00:22:59.593720 24418 solver.cpp:481]     Test net output #1: loss = 1.06661 (\* 1 = 1.06661 loss)
I0728 00:22:59.593730 24418 solver.cpp:411] Optimization Done.
I0728 00:22:59.593740 24418 caffe.cpp:187] Optimization Done.

I wonder if this accuracy given by the training process the final result? Do I need to test the trained model by running another program?

**Comment by yjxiong** [comment]
Please use the scripts provided in either `action_python` or `action_matlab` to run video-level prediction.
